client/src/css/structure.css

```diff
@@ -754,3 +754,11 @@ growl-notification.fading.ng-leave.ng-leave-active {
 .flex-header.static {
   box-shadow : none;
 }
+
+.icon-spacer {
+  margin-left : 18px;
+}
+
+.subtext {
```

Comment:
There is a helper class [text-muted](http://getbootstrap.com/css/#helper-classes-colors) which does something similar in bootstrap.  Would it be better than putting in a custom class?